### ISSUE TITLE
[Project] Wildfires

### ISSUE THREAD

**Issue by jessimckenzi** [jsoma/data-studio-projects#161]
# Pitch

Wildfires are getting worse, they say, and I wanted to take a look at the data

## Summary

I found a dataset going back to 1926 with data on the number of wildland fires annually, and the acreage burned annually. However, the data from before 1983 was not gathered with the same methodology as after 1983, so really the data only goes back to then. However, there might be an interesting story in why the older historical data shows more fires/year than the newer statistics? Over-reporting? Less well-equipped firefighting teams, resulting in more disastrous burns.

## Details

**Possible headline(s):**  35 Years of Wildland Fires; How America the Beautiful Burns

**Data set(s):**

Table of contents, of sorts: https://www.nifc.gov/fireInfo/fireInfo_statistics.html

Wildland fires and acres (1926-2017) https://www.nifc.gov/fireInfo/fireInfo_stats_totalFires.html

Wildfires larger than 100,000 acres (1997-2017) https://www.nifc.gov/fireInfo/fireInfo_stats_lgFires.html

There's obviously so much more on this, and if I were mapping I might be looking at the annual reports for each state, but they're in tables in PDFs, so getting the info is a big project all on its own

**Code repository:** 

https://github.com/jessimckenzi/data-studio/blob/master/02-fires/Wildfires.ipynb
	
**Possible problems/fears/questions:** 

As always, is this boring? What other kinds of data/info can I pull in to make it more interesting? What are the rules about correlation? Would comparing annual rainfall be kosher or not? One climate website graphed fires on forest service land with average temperature to show fires are more common in hotter years (not surprising!)

## Work so far

This is my full data set, the # of fires per year since 1926 [you'll notice a big drop between 1982 and 1983, when the methodology for data collection was standardized]:
![screen shot 2018-07-16 at 9 37 27 pm](https://user-images.githubusercontent.com/39416177/42794029-d677295c-894a-11e8-9d56-92b8853502dc.png)

Acreage burned every year since 1926:
![screen shot 2018-07-16 at 9 37 36 pm](https://user-images.githubusercontent.com/39416177/42794038-ded58b20-894a-11e8-83b2-57d21d4a7774.png)

Number of fires annually since 1983:
![screen shot 2018-07-16 at 9 37 46 pm](https://user-images.githubusercontent.com/39416177/42794051-f11b2b8c-894a-11e8-9468-ec9b4bcf4fc6.png)

Acreage burned annually since 1983:
![screen shot 2018-07-16 at 9 37 53 pm](https://user-images.githubusercontent.com/39416177/42794068-079e850c-894b-11e8-929b-8124b77649c7.png)

Avg acreage burned per fire since 1983:
![screen shot 2018-07-16 at 9 38 03 pm](https://user-images.githubusercontent.com/39416177/42794084-17a6c932-894b-11e8-9a7d-a486d4ede0ad.png)

This comes from a different data set on fires that burned more than 100,000 acres: I need to group them by year, maybe stack them? Anyway it's a mess right now:
![screen shot 2018-07-16 at 9 38 17 pm](https://user-images.githubusercontent.com/39416177/42794099-2fd4d47c-894b-11e8-9f94-6f81b857d348.png)

Some inspiration:
![screen shot 2018-07-16 at 10 48 10 pm](https://user-images.githubusercontent.com/39416177/42794110-3d99a876-894b-11e8-8871-73021bdc1091.png)

From: http://www.climatecentral.org/news/report-the-age-of-western-wildfires-14873

Background info/research:
https://grist.org/article/the-west-is-burning-and-its-barely-july/

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by jessimckenzi** [comment]
# Update
	
## I found data on el nino years and I overlaid that on the acreage burned per year graph to see if there was a correlation between the weather phenomenon and an increase in wildland fires.
	
![screen shot 2018-07-19 at 11 56 19 pm](https://user-images.githubusercontent.com/39416177/42982629-26b9649a-8bb0-11e8-9141-a49fe6db85a6.png)

  
## Any changes in direction or topic?

Not really, just the addition of El Nino data, which came from here:
http://origin.cpc.ncep.noaa.gov/products/analysis_monitoring/ensostuff/ONI_v5.php
	
## Problems/Questions
 
I myself can't really explain the graph, or if there's a story there, which is a problem, and I need to better convey what the line graph is attempting to show...
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by playfairbot** [comment]
Hi! I'm a little robot, here for a surprise inspection.

You need some feedback, let me summon @ElinaMak, @angelareplica, @Katerinavts for you

**Comment by angelareplica** [comment]
I love this idea! Overlaying graphs is a cool idea. There doesn't seem to be a strong correlation between El Nino and acreage burned per year though -- perhaps you'll be able to find a clearer relationship with other data. Also, it's a bit hard to see the line graph over the bars because both of the colors are so strong. I think a lighter bar color would help with that. Excited to see how this turns out!

**Comment by ElinaMak** [comment]
Very interesting topic! I also did the Natural disasters in USA topic and the different type of fires (and classified by their name is quite an issue when building a dataset / scraping etc). And always great to bring up the topic of correlation when dealing with different datasets. I had to google wildfires - El Ninio correlation because even from the "inspiration part", I am not sure if I could get any evident conclusion or pattern. But its a topic that requires some sort of expertise I guess...

**Comment by jessimckenzi** [comment]
# Update
	
## Your project content: 
	
Periods of El Niño conditions are associated in the United States with drier conditions in Hawaii, the Ohio Valley, Pacific Northwest and the Rocky Mountains. It often means warmer than average conditions for Alaska and the northern tier of the continental U.S., and cooler than average conditions for much of the southern tier of the U.S. I wanted to see if these drier, warmer conditions correlated with wildfires in the U.S.

From what I can tell, there might be some correlation, but not enough to explain the rise in wildfires over the past few decades—which shouldn't be surprising, when you consider that El Niño years are part of a complex weather and climate system, and doesn't account for drought or other on-the-ground conditions.

Title: El Niño years can coincide with spikes in wildland fires, but can't explain the rise of large fires over the past three decades

![fire-01-edited-01](https://user-images.githubusercontent.com/39416177/43156293-baf2d750-8f47-11e8-8de6-0b569afe43da.png)

  
## Any changes in direction or topic?

No, but I should try out a different dataset for comparison. Before I did that, I wanted to make this graph look better, even if the information isn't what I wanted.

HOWEVER, you will notice that the content of the graph is different! That's because I messed up and plotted the 1950s ONI data over the 1983 fire data, and didn't notice until I exported to AI and the line graph went off the Artboard.
	
## Problems/Questions
With the dual axis I couldn't figure out how to make edits to the chart with ax.blahblahblah. I wanted to edit the tick marks in matplotlib but couldn't figure it out so did it in Illustrator.

Cannot figure out why the two lines (which I drew in Illustrator because I struggled with the grid in matplotlib with a dual axis) look different! I copied one and pasted to make them identical, but it wasn't! Fussed with it more but still couldn't figure it out
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by jlstro** [comment]
Hey, great that you combined multiple data sources! To be honest, I struggle to understand what the values for each year actually mean - I guess a simple legend solves that. Design-wise, it might be nice to have less opacity in the line. It kinda looks like the line is cutting the bar, especially here:
![grafik](https://user-images.githubusercontent.com/3785873/43292866-69d882ba-9105-11e8-9211-6c23d9529ea6.png)

I personally would make the tick marks on the x-axis way smaller, but that is just my taste.

Other than these small design suggestions, I think that you did very well and came extremely close to the inspirational piece!

**Comment by hakantan** [comment]
Great that you went out and gathered additional data to look into. It is also cool that you went with dual axis, tho But from just looking at the graphs (and not reading the text) I feel kind of lost, because I don't know what is presented to me. After reading the text I can understand where you're going with this, but maybe a headline or something would help with that.

I would choose a different color than red. The way it is right now, I can't clearly see the graphs.

**Comment by jessimckenzi** [comment]
# Final

## Project visuals/text
 While the annual number of wildland fires in the United States has been relatively constant since the mid-1980s, over the last two decades they have become more devastating. It is no longer unusual for wildland fires to consume close to 10 million acres of land annually, according to records maintained by the National Interagency Fire Center.

While many have observed that the increase in size and severity of wildfires correlates with rising average temperatures, wildfires occur within a complex climate system. One study analyzed fire scars and tree rings dating back to 1550 and found that large fires in the southwestern United States are more common during La Niña events, which are associated with lower precipitation in that region. The pattern is reversed in the northwest, which have wetter than usual conditions during La Niña events.

The relationship between wildfires and the El Niño–Southern Oscillation climate pattern (ENSO), if it exists, is a complicated one. Simon Wang, a climate researcher at Utah State University, writes that “a wet year reduces fires while increasing vegetation growth, but then the increased vegetation dries out in subsequent dry years.” Fires that begin during dry years then have more fuel to consume.

“If the wet-dry cycle amplifies in the future, as Stevenson et al. (2012) suggest it might, more vegetation will grow in the wetter years and, subsequently, provide more fire fuel during the drought years,” Wang concludes. “The potential and extent of wildfire will continue to increase as a result.”


Annual wildland fires by acreage burned and the ENSO climate pattern (1983 - 2017)

Without more reliable and consistently gathered data on wildland fires, the complex relationship between ENSO and wildland fires is hard to see here.

![fire-01-edited-01](https://user-images.githubusercontent.com/39416177/43432787-8b0a67ba-9442-11e8-82c7-15a63a514efb.png)



Annual wildland fires by acreage burned and average annual rainfall (1983-2015)

It appears as though fires may tend to be larger in years following heavier than usual precipitation.

![fire-02-edited-01](https://user-images.githubusercontent.com/39416177/43432788-8b204f8a-9442-11e8-9c04-68cba5789e9d.png)


Annual wildland fires by acreage burned and average annual temperature (1983-2015)

The clearest correlation with the increase in large wildland fires is rising average annual temperature.

![fire-03-edited-01](https://user-images.githubusercontent.com/39416177/43432789-8b330558-9442-11e8-9818-a5b22df2c322.png)


## Details

**Headline:** 
10 Million Acres and More
**Published website version:**
https://jessimckenzi.github.io/fires/
**Code repository:**
https://github.com/jessimckenzi/data-studio/tree/master/02-fires
**Final data set(s):** 
https://www.nifc.gov/fireInfo/fireInfo_stats_totalFires.html
http://origin.cpc.ncep.noaa.gov/products/analysis_monitoring/ensostuff/ONI_v5.php
http://sdwebx.worldbank.org/climateportal/index.cfm?page=downscaled_data_download&menu=historical

## What did you find to be the most difficult part of this project?
Understanding and interpreting the science; trying to determine correlation

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I mean, as someone fairly new to climate reporting, I'm ok with it, but it's not exactly breaking news.

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
